src/main/java/ch/njol/skript/expressions/ExprDateInTimezone.java

```diff
@@ -0,0 +1,106 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.config.Node;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Example;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.skript.util.Date;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+import org.skriptlang.skript.log.runtime.SyntaxRuntimeErrorProducer;
+
+import java.time.DateTimeException;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+
+@Name("Date in Timezone")
+@Description({
+	"Returns a date in the specified timezone. Note that the result date might not be equal to the input date.",
+	"Use <a href='#ExprAllTimezones'>all timezones</a> to get a list of valid timezones."
+})
+@Example("""
+	set {_date} to now in timezone "Europe/Istanbul"
+	set {_clock} to {_date} formatted as "kk:mm"
+	send "It is currently %{_clock}% in Istanbul!" to player
+	""")
+@Since("INSERT VERSION")
+public class ExprDateInTimezone extends SimpleExpression<Date> implements SyntaxRuntimeErrorProducer {
```

Comment:
```suggestion
public class ExprDateInTimezone extends SimpleExpression<Date> {
```
`SimpleExpression` now auto implements `SREP` from Sovde's `preInit` PR.